spacy/tests/test_gold.py

```diff
@@ -68,3 +71,23 @@ def test_gold_ner_missing_tags(en_tokenizer):
     doc = en_tokenizer("I flew to Silicon Valley via London.")
     biluo_tags = [None, "O", "O", "B-LOC", "L-LOC", "O", "U-GPE", "O"]
     gold = GoldParse(doc, entities=biluo_tags)  # noqa: F841
+
+
+def test_docs_to_json_roundtrip(tmpdir):
+    text = "I flew to Silicon Valley via London."
+    nlp = English()
+    doc = nlp(text)
+    doc[0].is_sent_start = True
+    for i in range(1, len(doc)):
+        doc[i].is_sent_start = False
+
+    json_str = srsly.json_dumps(docs_to_json(doc))
+    json_file = tmpdir.mkdir("docs_to_json_roundtrip").join("flew.json")
+    with open(json_file, "w") as fileh:
```

Comment:
I think the tempdir stuff is what makes the tests fail on some platforms. We have a `tests.util.make_tempdir` contextmanager that should hopefully help with this and make the test more consistent.

Reply:
Thanks, I didn't think to check that. I rewrote it to use make_tempdir and look more like some of the existing tests that write temp files.

Reply:
Thanks, looks like this did the trick! And no worries – most of that stuff is super arbitrary.